### PR TITLE
Installation goes brrrr

### PR DESCRIPTION
**Pull request by nickvergessen**
Reduced installation time by ~50%

### CODE REVIEW

File: lib/private/Installer.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function installApp(string $appId, bool $forceEnable = false): string {
 			}
 		} else {
 			$ms = new \OC\DB\MigrationService($info['id'], \OC::$server->getDatabaseConnection());
-			$ms->migrate();
+			$ms->migrate('latest', true);
```
Comment (nickvergessen) [inline lib/private/Installer.php:157]: When we install, we have no data, so we can skip the before and after migration parts which saves some time

**Comment by MorrisJobke** [comment]
Really nice 👍 

Bildschirmfoto 2020-11-11 um 12 14 09
Bildschirmfoto 2020-11-11 um 12 14 19

**Review by MorrisJobke — APPROVED** [review]
Code makes sense, tested and works 👍

**Review by juliusknorr — APPROVED** [review]
:rocket:

**Comment by ChristophWurst** [comment]
Brrrreaks apps as `app:enable` doesn't migrate anymore

**Comment by MorrisJobke** [comment]
> Brrrreaks apps as `app:enable` doesn't migrate anymore

Why don't we have tests for this 🙈

**Comment by ChristophWurst** [comment]
Because it would fail

**Pull request merged**